data/bangs.json

```diff
@@ -67385,17 +67385,17 @@
   },
   {
     "s": "Openstreetmap forum Netherlands",
-    "d": "forum.openstreetmap.org",
+    "d": "community.openstreetmap.org",
     "t": "osmforumnl",
-    "u": "https://forum.openstreetmap.org/search.php?action=search&keywords={{{s}}}&forums[]=12",
+    "u": "https://community.openstreetmap.org/search?q={{{s}}} #communities:nl",
```

Comment:
this has to be URI encoded for it to work, so you can just add a `"skip_tests": true` field to this bang

Reply:
Alternatively, I could remove space and `#` from being checked in the URI encoded tests

Reply:
I don't know what's best tbh, you're the expert. Feel free to make a decision on what you think is best and change accordingly